app/controllers/concerns/login_authenticator.rb

```diff
@@ -0,0 +1,25 @@
+module LoginAuthenticator
```

Comment:
make sure that logged in user can't go to login/sign-up pages

Reply:
same for reset_passwords

Reply:
done